dracut/30ignition/ignition-check-neednet.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ -f /run/ignition/neednet ]; then
+    echo "rd.neednet=1" > /etc/cmdline.d/40-ignition-neednet.conf
+
+    # Hack: we need to rerun the NM cmdline hook because we run after
+    # dracut-cmdline.service because we need udev. We should be able to move
+    # away from this once we run NM as a systemd unit. See also:
+    # https://github.com/coreos/fedora-coreos-config/pull/346#discussion_r409843428
+    set +euo pipefail
+    . /usr/lib/dracut-lib.sh
+    . /usr/lib/dracut/hooks/cmdline/99-nm-config.sh
+    set -euo pipefail
```

Comment:
Is there a way we could detect to see if networking config was already written out (i.e. `rd.neednet=1` was set before `dracut-cmdline.service` ran) and just do nothing here? I'm not sure if running `/usr/lib/dracut/hooks/cmdline/99-nm-config.sh` multiple times has any unintended consequences.

Reply:
Thanks, fixed!